RandLAPACK/drivers/rl_bqrrp_gpu.hh

```diff
@@ -358,8 +358,9 @@ int BQRRP_GPU<T, RNG>::call(
             lapack::getrf_work_size_bytes(cols, sampling_dimension, A_sk_trans, n, &d_size_getrf, &h_size_getrf, lapack_queue);
 
             d_work_getrf = blas::device_malloc< char >( d_size_getrf, lapack_queue );
-            std::vector<char> h_work_getrf_vector( h_size_getrf );
-            h_work_getrf = h_work_getrf_vector.data();
+            // Below shoudl not be necessary
+            //std::vector<char> h_work_getrf_vector( h_size_getrf );
+            //h_work_getrf = h_work_getrf_vector.data();
```

Comment:
Either remove this commented-out code or have the comment include an explanation for keeping the commented-out code.

Reply:
Removed